docs/sources/send-data/otel/native_otlp_vs_loki_exporter.md

```diff
@@ -0,0 +1,95 @@
+
+## What has changed?
+
+### Formatting of logs
+
+[LokiExporter's README](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/lokiexporter/README.md) explains in detail the formatting of OTel logs ingested via LokiExporter, while Loki’s native OTel log ingestion endpoint is described in detail in its [documentation](https://grafana.com/docs/loki/<LOKI_VERSION>/send-data/otel/). Here is a summary of how the logs are formatted between the two:
+
+**LokiExporter**
+
+- Index Labels:
+  - It supports label control via hints set by the OTel client/collector.
+  - Default labels:
+    - job=service.namespace/service.name
+    - instance=service.instance.id
+    - exporter=OTLP
+    - level=severity
+- Log Body: Encodes log records and attributes in json(default) or logfmt format.
+
+**Loki’s native OTel log ingestion endpoint**
+
+- Index Labels:
+  - It supports label control via per-tenant OTLP configuration in Loki.
+  - By default, it picks some pre-configured resource attributes as index labels as explained [here](http://localhost:3002/docs/loki/<LOKI_VERSION>/send-data/otel/#format-considerations).
+- LogLine: Stringified LogRecord.Body.
+- Structured Metadata: Anything not stored in Index labels and LogLine gets stored as Structured Metadata.
+
+### Sample Log
+
+Let us take the following sample OTel log and see how it would look after ingestion in both formats with the default configuration:
+- Resource Attributes:
+  - service.name: “auth”
+  - service.namespace: “dev”
+  - service.kind: “app”
+- Log Record:
+  - Timestamp: 1715247552000000000
+  - Body: “user logged in”
+  - Severity Text: “INFO”
+  - Attributes:
+    - email: “foo@bar.com”
+
+**Ingested with LokiExporter:**
+
+- Index Labels:
+  - job=”dev/auth”
+  - exporter=”OTLP”
+- Log Body: `{"body":"user logged in","severity":"INFO","attributes":{"email":"foo@bar.com"}, "resources":{"service.kind":"app","service.name":"auth","service.namespace":"dev"}}`
+
+**Ingested with Loki’s native OTel endpoint:**
+
+- Index Labels:
+  - service_name=”auth”
+  - service_namespace=”dev”
+- Log Body: “user logged in”
+- Structured Metadata:
+  - service_kind: “app”
+  - severity_text: “INFO”
+  - email: “foo@bar.com”
+
+### Querying experience
+
+Taking the above-ingested log line, let us look at how the querying experience would look between the two considering various scenarios:
```

Comment:
I still want to rewrite this somehow, for example, "Lets look at how you would need to write different queries to return the same results depending on how you ingested your OTel logs into Loki.

Reply:
I have added some text under querying examples and more query scenarios. Please let me know if we need to improve it further.